syft/workers/base.py

```diff
@@ -28,13 +28,16 @@
 from syft.messaging.message import ForceObjectDeleteMessage
 from syft.messaging.message import GetShapeMessage
 from syft.messaging.message import IsNoneMessage
+from syft.messaging.message import CryptenInit
 from syft.messaging.message import Message
 from syft.messaging.message import ObjectMessage
 from syft.messaging.message import ObjectRequestMessage
 from syft.messaging.message import PlanCommandMessage
 from syft.messaging.message import SearchMessage
 from syft.workers.abstract import AbstractWorker
 
+from syft.frameworks.crypten import run_party
```

Comment:
`BaseWorker` intentionally doesn't know anything about frameworks, so would like to find a way to avoid letting that knowledge leak in. One possibility would be to create a `CryptenWorker` that overrides or adds to the message dispatch table.

Reply:
Good idea :)

Reply:
Alternatively, how bad would that be to put the import in the `def run_crypten_party` ?
I've already seen things like this but I don't know how evil it is

Reply:
I think the problem is that we are exposing some methods that are not necessarily needed by the ```VirtualWorker``` (@karlhigley this is the reason?)